docs/_static/custom.css

```
@@ -1,395 +1,738 @@
-    margin-left: 1rem !important; /* Small gap between Link and Search */
-    min-width: auto;
+  margin-left: 1rem !important;
+  min-width: auto;
 }
-
-/* Navbar Links */
 .navbar-nav .nav-link {
-    font-family: "Satoshi", sans-serif;
-    color: var(--orion-blue) !important;
-    font-weight: 500;
+  font-family: "Satoshi", sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Satoshi" (font-family-name-quotes)

(font-family-name-quotes)